tests/sb-util.test.ts

```diff
@@ -89,11 +89,13 @@ describe('ScratchProject class --------------------', () => {
 			test('to query sprites where value is a string with double quotes', () => {
 				const sprite = sp.sprites('[name="Sprite1"]');;
 				expect(sprite).toBeInstanceOf(SpriteCollection);
+				expect(sprites.first()).toBeInstanceOf(Sprite);
 			});
 
 			test('to query sprites where value is a string with single quotes', () => {
 				const sprite = sp.sprites("[name='Sprite1']");;
 				expect(sprite).toBeInstanceOf(SpriteCollection);
+				expect(sprites.first()).toBeInstanceOf(Sprite);
```

Comment:
Same here.

Reply:
Same here, with the debugging line hypothesis